assets/questions/i18n.js

```diff
@@ -70,9 +74,9 @@ def createListOfUniqueStrings(atom, size):
       expectedOutput: 'f8p'
     }],
     buggyOutputTests: [{
-      buggyFunction: 'forgetLastLetter',
+      buggyFunction: 'AuxiliaryCode.forgetLastLetter',
       messages: [
-        "It looks like your output (%s) doesn't match our expected output (%s).",
+        "Try running your code on 'word'. What's the result?.",
```

Comment:
Do you want to say something like "in your head" or "on a piece of paper" here, so that they don't just click the run button?